package.json

```diff
@@ -51,6 +51,7 @@
     "@trpc/server": "^10.37.1",
     "drizzle-orm": "^0.29.3",
     "drizzle-zod": "^0.5.1",
+    "file-loader": "^6.2.0",
```

Comment:
I don't see the package in the cache. Do we need it? If so try running `yarn install` so it's added to the cache. If we don't delete this package